File: dsa/cpp/funtion/call_by_value_and_refrence.cpp
#include<iostream>
using namespace std;

int changevalue(int a)
{
	a=10;
	cout<<"inside fuction a:"<<a<<endl;    
    return a;
}




int main()
{
int a=5;
cout<<"the number before:"<<a<<endl;
changevalue(a);
cout<<"after fuction call:"<<a;
}
	

File: dsa/java/basics/funtion/CallByValueAndRefrence.java
public class CallByValueAndRefrence {
    static int changeValue(int a) {
        a = 10;
        System.out.println("inside function a: " + a);
        return a;
    }

    public static void main(String[] args) {
        int a = 5;
        System.out.println("the number before: " + a);
        changeValue(a);
        System.out.println("after function call: " + a);
    }
}
